reframe/frontend/cli.py

```diff
@@ -586,10 +584,44 @@ def main():
     printer.debug(format_env(options.env_vars))
 
     # Setup the check loader
+    if options.restore_session is not None:
+        # We need to load the failed checks only from a report
+        if options.restore_session:
+            filename = options.restore_session
+        else:
+            filename = runreport.next_report_filename(
+                osext.expandvars(site_config.get('general/0/report_file')),
+                new=False
+            )
+
+        report = runreport.load_report(filename)
+        check_search_path = list(report.slice('filename', unique=True))
+        check_search_recursive = False
+
+        # If `-c` or `-R` are passed explicitly outside the configuration
+        # file, override the values set from the report file
+        if site_config.is_sticky_option('general/check_search_path'):
```

Comment:
Here is would be good to print that you are overriding the search path of the report with the one passed from the configuration. In my case for example I had the `RFM_CHECK_SEARCH_PATH` because I have loaded the `reframe` module so that search path was always overriding the one from the report.

Reply:
I am planning to print something at "info" level, because it's not necessarily a bad thing or undesirable, so as to give a warning. What do you think?

Reply:
I will make it warning, after all, because otherwise it doesn't stand out...